src/Components/Activities/Table/List.jsx

```diff
@@ -0,0 +1,61 @@
+import { useState, useEffect } from 'react';
```

Comment:
```suggestion
import React, { useState, useEffect } from 'react';
```
You should import React, like this for example